python/ribasim/ribasim/config.py

```diff
@@ -3,17 +3,51 @@
 
 from __future__ import annotations
 
-from datetime import datetime
 from typing import List, Optional, Union
 
-from pydantic import BaseModel, Field
+from pydantic import BaseModel
+
+from ribasim.input_base import NodeModel, TableModel
+from ribasim.schemas import (
+    BasinProfileSchema,
+    BasinStateSchema,
+    BasinStaticSchema,
+    BasinTimeSchema,
+    DiscreteControlConditionSchema,
+    DiscreteControlLogicSchema,
+    FlowBoundaryStaticSchema,
+    FlowBoundaryTimeSchema,
+    FractionalFlowStaticSchema,
+    LevelBoundaryStaticSchema,
+    LevelBoundaryTimeSchema,
+    LinearResistanceStaticSchema,
+    ManningResistanceStaticSchema,
+    OutletStaticSchema,
+    PidControlStaticSchema,
+    PidControlTimeSchema,
+    PumpStaticSchema,
+    TabulatedRatingCurveStaticSchema,
+    TabulatedRatingCurveTimeSchema,
+    TerminalStaticSchema,
+    UserStaticSchema,
+    UserTimeSchema,
+)
 
 
 class Allocation(BaseModel):
     timestep: Optional[float] = None
     use_allocation: bool = False
 
 
+class Results(BaseModel):
+    basin: str = "results/basin.arrow"
+    flow: str = "results/flow.arrow"
+    control: str = "results/control.arrow"
+    outstate: Optional[str] = None
+    compression: str = "zstd"
```

Comment:
Could this be a `str enum` instead?

Reply:
I made it to an Enum (I did read about that Literal is faster, but that doesn't matter here).